7_react/react1/week1/todo-app/src/App.jsx

```diff
@@ -0,0 +1,27 @@
+import TodoList from "./components/TodoList/TodoList";
+
+function App() {
+const todos = [
```

Comment:
Hey @GizemSavci! No worries I am not going to block the HW :D, just to keep you informed as I mentioned this to others as well. Leaving static variables as `todos` array is considered bad practice, I know that you haven't learned about React's lifecycle methods yet, but take this just as a fact for now:) Nevertheless, the fix is super simple. Just move the variable outside of the render(React component) like this. 
```
const todos = [..your data]
function App() {
```
Good Job!👏

Reply:
Thanks @BrantalikP I implemented that🌸